src/app/(safespace)/space/passwort/strength/page.tsx

```diff
@@ -26,11 +25,37 @@ export default function PasswordStrength() {
   const [animateShake, setAnimateShake] = useState(false);
   const [animatePulse, setAnimatePulse] = useState(false);
 
+  useEffect(() => {
+    const fetchHighScore = async () => {
+      const context = new PersistUserService();
+      const loadedHighScore = await context.getHighScore("PASSWORD_STRENGTH");
+      setHighscore(loadedHighScore);
+    };
+
+    fetchHighScore();
+  }, [highscore]);
+
+  useEffect(() => {
+    if (gameStarted) {
+      passwordAnimation(currentQuestion.password);
+    }
+  }, [gameStarted, currentQuestionIndex, currentQuestion.password]);
+
+  useEffect(() => {
+    let storedUser = null;
+    console.debug("user", storedUser);
```

Comment:
why?? we dont need the storeduser object in general if you apply my previous comment (force user to create acc via redirect if no userid exist in loclastorage). so please remove storedUser